packages/macro/src/index.macro.ts

```diff
@@ -4,6 +4,8 @@ import * as path from "path";
 import * as fs from "fs";
 import * as t from "@babel/types";
 
+export type Dependency = number | string | { uniqueId: string; }
```

Comment:
@Romex91 Do you have a better idea to share this type between `macro` and `runtime` packages? Or it's OK to duplicate this type for now?

Reply:
@Romex91 Done